<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>querystu</class>
 <widget class="QDialog" name="querystu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>631</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,8,2">
      <item>
       <widget class="QComboBox" name="cbb_searchway">
        <item>
         <property name="text">
          <string>请选择查询方式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按姓名查询</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按序号查询</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_cnt"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
